db/migrate/20190805204523_drop_deprecated_columns.rb

```diff
@@ -0,0 +1,47 @@
+class DropDeprecatedColumns < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :pipeline_run_stages, :failed_jobs
+
+    remove_column :host_genomes, :skip_deutero_filter
+
+    remove_column :metadata_fields, :validation_type
+
+    remove_column :pipeline_runs, :version
+    remove_column :pipeline_runs, :ready_step
+    # TODO: (gdingle): verify these pipeline_runs cols are all superseded buy pipeline_run_stages cols
+    remove_column :pipeline_runs, :job_id
+    remove_column :pipeline_runs, :command
+    remove_column :pipeline_runs, :command_stdout
+    remove_column :pipeline_runs, :command_error
+    remove_column :pipeline_runs, :command_status
+    remove_column :pipeline_runs, :job_description
+    remove_column :pipeline_runs, :job_log_id
+    remove_column :pipeline_runs, :postprocess_status
+    remove_column :pipeline_runs, :assembled_taxids
+
+    # TODO: (gdingle): check the 2% where this is present
+    remove_column :taxon_counts, :percent_concordant
```

Comment:
not sure about these *_concordant ones

Reply:
@katrinakalantar @cdebourcy  these concordance values have not set in `taxon_counts` since `2018-03-15`. Do you know what happened and whether we should bring them back?

Reply:
percent_concordant used to be a value in the report, but we started hiding it (https://github.com/chanzuckerberg/idseq-web/blob/0b9078947e757302ccbd8c746dc9c3f3da2b429d/app/assets/src/components/PipelineSampleReport.jsx#L129), and now it's no longer being computed in the backend either. I believe the column can be removed from the taxon_counts table, but in that case all references to it need to be removed throughout the codebase:

![image](https://user-images.githubusercontent.com/16944816/62746863-97bfa480-ba06-11e9-9142-d8b1d53ebb4c.png)

Reply:
> This list looks right to me (unless otherwise commented). I think the column values speak for themselves and :failed_jobs, :skip_deutero_filter, etc. are surely legacy fields..

I disagree. `skip_deutero_filter` is a column we need. It tells you something about the host genome. For mammals, deuterostome filtering should be performed. For mosquitoes, it should not be performed, because mosquitoes are not deuterostomes. That's why Mosquito host is currently the only one that has skip_deutero_filter == 1. This value is being actively used in the pipeline.

`failed_jobs` is being used in the codebase: https://github.com/chanzuckerberg/idseq-web/blob/cd4311e3b3d1e42eacdc267c261f84331ad8cc56/app/models/pipeline_run_stage.rb#L145
It's possible that that code path is broken. In that case, we first need to get to the bottom of that behavior before we can remove the column.

Reply:
thanks, I removed all the concordance.